src/nvme/ioctl.c

```diff
@@ -1597,12 +1597,12 @@ int nvme_set_var_size_tags(__u32 *cmd_dw2, __u32 *cmd_dw3, __u32 *cmd_dw14,
 	__u32 cdw2 = 0, cdw3 = 0, cdw14;
 
 	switch (pif) {
-	// 16b Protection Information
+	/* 16b Protection Information */
```

Comment:
Please merge these code fixups into previous patch.

Reply:
Fixed this in original commit.